### PR TITLE
Slack v2 doc - add oproxy information

### PR DESCRIPTION
**Pull request by liorblob**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23635

## Description
Add more information regarding what we send to oproxy in Slack.

### CODE REVIEW

File: Packs/Slack/Integrations/Slack/README.md [hunk 1]
```diff
@@ -748,7 +748,23 @@ mirror incident-id
...
 The integration also allows some level of interactivity. When a user interacts with an element in a Slack message, Slack sends a request with the relevant information. 
-This request is processed and stored by a dedicated endpoint outside of Demisto in the address: <code>https://oproxy.demisto.ninja</code>. What the integration currently allows is polling this endpoint for user interactions that contain entitlement strings(these are used to perform actions in Demisto by outside users, see the <code>SlackAsk</code> script for an example). What that means is that in order to enable interactivity using the integration, connection to this endpoint has to be enabled.</span>
+This request is processed and stored by a dedicated endpoint outside of Demisto in the address: <code>https://oproxy.demisto.ninja</code>.
```
Comment (Itay4) [inline Packs/Slack/Integrations/Slack/README.md:751]: `Demisto` -> `Cortex XSOAR`

File: Packs/Slack/Integrations/Slack/README.md [hunk 2]
```diff
@@ -748,7 +748,23 @@ mirror incident-id
...
+What the integration currently allows is polling this endpoint for user interactions that contain entitlement strings(these are used to perform actions in Demisto by external users, see the <code>SlackAsk</code> script for an example).
```
Comment (Itay4) [inline Packs/Slack/Integrations/Slack/README.md:752]: lets add link to `SlackAsk`

Comment (liorblob) [reply Packs/Slack/Integrations/Slack/README.md:752]: link to github or dev docs?

Comment (Itay4) [reply Packs/Slack/Integrations/Slack/README.md:752]: github

**Comment by kirbles19** [comment]
@liorblob Done.

**Pull request merged**